Core/include/Acts/Seeding/SeedFilter.ipp

```diff
@@ -65,34 +65,44 @@ void SeedFilter<external_spacepoint_t>::filterSeeds_2SpFixed(
               });
   }
 
-  for (auto& i : idx) {
+  size_t startCompTopLoopIdx = 0;
+  // loop over top SPs and other compatible top SP candidates
```

Comment:
Is there an important assumption about the SPs being sorted by phi used here? May be worth mentioning in the comment.